scripts/install-minio.sh

```diff
@@ -47,7 +47,12 @@ install_yum_pkgs() {
 install_brew_pkgs() {
   # use minio repo rather than cask,
   # as recommended by minio
-  brew install minio/stable/minio
+  #brew install minio/stable/minio
+
+  # Use direct installation due to failed download from homebrew
+  curl -O https://dl.min.io/server/minio/release/darwin-amd64/minio
```

Comment:
Should we use a fixed release?

Reply:
I'm inclined to stay with rolling since that's how homebrew was/is (and I'd like to switch back to homebrew when the CDN or 🤷‍♂️  issue is fixed).